use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,292 @@
+# How To Pass a Physical NIC Into a Kata Container
+![](https://github.com/Yuval-Ai/documentation/blob/doc-create-NIC-passthrough-and-Kata/use-cases/images/NIC%20passthrough%20Diagram.png)
+
+
+In this guide we walk through the process of passing a physical NIC into a KC (Kata Container). The "usual" way that a KC is wired for communication is through a series of bridges and virtual NICs as can be seen in the Image above (marked as default). 
+
+For some use cases, the container demands a direct link to the physical port of the host device, for example in a situation were the container is required to route high BW traffic without having support for acceleration such as SR-IOV.
+
+#### In this guide:
+
+* [Before you start (Restrictions, Requirements and Assumptions)](#before-you-start-restrictions-requirements-and-assumptions)
+* [Pass a physical NIC into a Kata Container](#Pass-a-physical-NIC-into-a-Kata-Container)
+  * [Part 1 – The Host](#part-1--the-host)
+  * [Part 2 – The Container](#part-2--the-container)
+* [Test Your Setup](#Test-Your-Setup)
+
+## Before you start (Restrictions, Requirements and Assumptions)  
+
+The method described in this guide relays on your system [supporting  IOMMU.]( https://en.wikipedia.org/wiki/Input–output_memory_management_unit#Published_specifications) before you start, make sure that your system has this attribute. If it does not, you should look for a different solution then the one presented here. 
+
+If you are uncertain about your system’s support, the first two steps in [Part 1 – The Host](# Part-1-–-The-Host) should help you figure this out. 
+
+ In my setup, I used a server with *Intel(R) Xeon(R) CPU D-1548 @ 2.00GHz* and [support for VT-d](https://en.wikipedia.org/wiki/X86_virtualization#Intel-VT-d), Operated by Ubuntu 18.04.3 LTS. The general steps proposed is this guide should match other hardware and OS with some changes.  
```

Comment:
I would make this more generic - rather than "In my setup" you could say, "This guide was tested using ...".